src/Components/Web.JS/src/Rendering/JSRootComponents.ts

```diff
@@ -7,6 +7,12 @@ let nextPendingDynamicRootComponentIdentifier = 0;
 type ComponentParameters = object | null | undefined;
 
 let manager: DotNet.DotNetObject | undefined;
+let initializationCallbacks : { failedToInitialize?: (reason?: any) => void, initialized?: () => void } = {};
```

Comment:
We don't need to capture a "reject" handler (`failedToInitialize`) because it will never happen (see comment below), so this could be simplified slightly.